python/python/glide/async_commands/core.py

```diff
@@ -4376,6 +4376,65 @@ async def zintercard(self, keys: List[str], limit: Optional[int] = None) -> int:
             await self._execute_command(RequestType.ZInterCard, args),
         )
 
+    async def zrevrank(self, key: str, member: str) -> Optional[int]:
```

Comment:
can you move this command to be adjacent to zrank? They are practically identical commands.